clients/js/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015"],
+  "sourceMaps": false
+}
```

Comment:
To translate recent js code in a code that'll be understood by older browsers: https://babeljs.io/